docker/build/dev-requirements.txt

```diff
@@ -27,7 +28,7 @@ ipcalc==1.99.0
 isodate==0.5.4
 Jinja2==2.9.6
 js.jqgrid==4.4.4
-js.jquery==1.9.1  # Fixed for admin interface
+js.jquery==1.7.1  # Fixed for admin interface
```

Comment:
yes it should be everywhere 1.7, actually it's at two places 1.7, and one 1.9 ...